### PR TITLE
Fix remote ssh login

### PR DESCRIPTION
**Pull request by kravciak**
Fix slow response problem
https://openqa.suse.de/tests/384581/modules/welcome/steps/3

Depends on:
https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/122

### CODE REVIEW

**Comment by sysrich** [comment]
Just to make sure I am not misunderstanding the intent of this PR .. are you trying to resolve ssh login, not 'logging' ? ;)

**Comment by kravciak** [comment]
Yes, sorry - I had that strange feeling when I submitted, but... :)

File: tests/remote/remote_master.pm [hunk 1]
```diff
@@ -39,12 +40,14 @@ sub run() {
     elsif (check_var("REMOTE_MASTER", "ssh")) {
         $slave_ip = "10.0.2.11";
         select_console 'user-console';
+        clear_console;
+
         type_string "ssh root\@$slave_ip\n";
-        sleep 2;
+        assert_screen "ssh-login";
```
Comment (sysrich) [inline tests/remote/remote_master.pm]: Are we 100% sure that ssh will never know the fingerprint of the slave?

Comment (kravciak) [inline tests/remote/remote_master.pm]: slave is new installation - at this step I think fingerprint should be unknown

**Comment by okurz** [comment]
this should be the "golden example" why "sleep X" in testcode is evil.

**Comment by kravciak** [comment]
As I wrote in description, please also merge:
https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/122

**Comment by Soulofdestiny** [comment]
@kravciak needles merged

**Pull request merged**